common/constants.py

```diff
@@ -0,0 +1,3 @@
+import os
+DEV_ENV = os.environ['DEV'] == 'True'
```

Comment:
I think this is redundant but ok...

I prefer using `os.environ['DEV']` to `constants.DEV_ENV`.

Also because it's an environment  _variable_ and not a _constant_ 😛

Reply:
You are right, although it might be a good idea to have a constant file, just in case .